metrics/cli/lib/services/common/mixin/authorization.dart

```diff
@@ -0,0 +1,11 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+/// A mixin that provides common methods for setting the authorization up.
+mixin Authorization {
```

Comment:
🏗️ 
Not sure this should be a mixin as mixins are most likely used to share code between classes. In our case, we can pass the auth to adapters so they can explicitly use the given value where necessary. Otherwise, we should think about creating the `AuthService` interface as we have for the `InfoService`.